api/status.go

```diff
@@ -28,6 +28,7 @@ func InitStatus() {
 	l4g.Debug(utils.T("api.status.init.debug"))
 
 	BaseRoutes.Users.Handle("/status", ApiUserRequiredActivity(getStatusesHttp, false)).Methods("GET")
+	BaseRoutes.Users.Handle("/team_statuses", ApiUserRequiredActivity(getTeamStatusesHttp, false)).Methods("POST")
```

Comment:
Should this maybe be under BaseRoutes.NeedsTeam? That way the team ID will already be included in the URL and won't need to be passed in the body. Also, it could get a GET call then as well.